### PR TITLE
Add new setup guides, updated names, adjust layout

### PR DESCRIPTION
**Pull request by OhanNanen-MS**
Updated doc with new setup guides. Also updated names, and access instructions based on changes within the Microsoft 365 admin center. Changed page layout based on feedback.

### CODE REVIEW

**Comment by JoeDavies-MSFT** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 1]
```diff
@@ -20,47 +20,22 @@ search.appverid:
 - MET150
 - BCS160
 ms.assetid: 165f46e8-3533-4d76-be57-97f81ebd40f2
-description: "Accelerate your planning and configuration of Microsoft 365 or Office 365 with setup guides."
+description: "Get step by step tools to plan, migrate and implement the features in your tenant’s licenses. Find a guide to set up a service or an app you need to run."
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
description: "Get step-by-step tools to plan, migrate, and implement the features in your tenant’s licenses. Find a guide to set up a service or an app you need to run."
```

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 2]
```diff
@@ -20,47 +20,22 @@ search.appverid:
...
 
-### Email setup advisor
+### Email setup guide
 
-The [Email setup advisor](https://aka.ms/office365setup) provides you with the step-by-step guidance needed for configuring Exchange Online for your organization. This includes setting up new email accounts, migrating email, and configuring email protection. For a successful email set up, use this advisor and you'll receive the recommended migration method based on your organization's current mail system, the number of mailboxes being migrated, and how you want to manage users and their access.
+The [Email setup guide](https://aka.ms/office365setup) provides you with the step-by-step guidance needed for configuring Exchange Online for your organization. This includes setting up new email accounts, migrating email, and configuring email protection. For a successful email set up, use this advisor and you'll receive the recommended migration method based on your organization's current mail system, the number of mailboxes being migrated, and how you want to manage users and their access.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
The [Email setup guide](https://aka.ms/office365setup) provides you with the step-by-step guidance needed for configuring Exchange Online for your organization. This includes setting up new email accounts, migrating email, and configuring email protection. For a successful email setup, use this advisor and you'll receive the recommended migration method based on your organization's current mail system, the number of mailboxes being migrated, and how you want to manage users and their access.
```

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 3]
```diff
@@ -109,9 +88,9 @@ The [Azure AD setup guide](https://aka.ms/aadpguidance) provides information to
 
 It also includes essential information on enabling self-service password resets, conditional access and integrated third-party sign-on including optional advanced identity protection and user provisioning automation.
 
-### Sync users from your org’s directory
+### Sync users from your Windows Server Active directory
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
### Sync users from your Windows Server Active Directory
```

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 4]
```diff
@@ -109,9 +88,9 @@ The [Azure AD setup guide](https://aka.ms/aadpguidance) provides information to
...
 
-The [Sync users from your org’s directory wizard](https://aka.ms/directorysyncsetup) walks you through turning on directory synchronization. This brings your on-premises and cloud identities together for easier access and simplified management. Unlock new capabilities, like single sign-on, self-service options, automatic account provisioning, conditional access controls, and compliance policies. This ensures that your users have access to the resources they need from anywhere.
+The [Sync users from your Windows Server Active directory wizard](https://aka.ms/directorysyncsetup) walks you through turning on directory synchronization. This brings your on-premises and cloud identities together for easier access and simplified management. Unlock new capabilities, like single sign-on, self-service options, automatic account provisioning, conditional access controls, and compliance policies. This ensures that your users have access to the resources they need from anywhere.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
The [Sync users from your Windows Server Active Directory wizard](https://aka.ms/directorysyncsetup) walks you through turning on directory synchronization. This brings your on-premises and cloud identities together for easier access and simplified management. Unlock new capabilities, like single sign-on, self-service options, automatic account provisioning, conditional access controls, and compliance policies. This ensures that your users have access to the resources they need from anywhere.
```

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 5]
```diff
@@ -211,3 +198,33 @@ Use the [Deploy Office to remote users guide](https://aka.ms/officeremoteinstall
...
+2. Find the **Training & guides** card.
+
+   ![Training & guides card in the Microsoft 365 admin center](../media/setup-guides-for-microsoft-365/adminportal-trainingandguides.png)
+
+3. Select **Step-by-step guides**.
+
+   ![Screenshot of the Setup guidance page in the Microsoft 365 admin center](../media/setup-guides-for-microsoft-365/adminportal-setupguidance.png)
+
+>[!NOTE]
+>Tenant administrator permissions are required to access the Microsoft 365 admin center.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
> [!NOTE]
> Tenant administrator permissions are required to access the Microsoft 365 admin center.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @OhanNanen-MS, kindly commit the minor suggested changes. Many thanks.

**Comment by OhanNanen-MS** [comment]
> Copy review - @OhanNanen-MS, kindly commit the minor suggested changes. Many thanks.

@JohanFreelancer9 All changes are committed. Thanks!

**Pull request merged**